src/localization/de.js

```diff
@@ -19,7 +19,67 @@ export default {
     toAboutPage:      'Mehr lernen',
   },
 
-  aboutPage: {},
+  aboutPage: {
+    aboutPageHeader: 'Über dieses Cliff Effects Werkzeug',
+    whatForHeader:  'Wozu benutzt man dieses Werkzeug?',
+    whatForImportantNote: 'Wichtige Anmerkung: Diese Anwendung ist ein Minimum Viable Product ' + 
+    '(minimal überlebensfähiges Produkt).  Sie sollte nicht als das einzige Werkzeug ' + 
+    'benutzt werden, um die SNAP- oder Section-8- (oder irgendein anderes ' + 
+    'Sozialhilfeprogramm-) finanzielle Situation eines Klienten zu verstehen.',
+    whatFor1A:  'Dieses Werkzeug kann zeigen helfen, wie eine Einkommensveränderung ' + 
+    'sich auf die Sozialhilfe von SNAP und Section-8-Wohnungsgutschein auswirkt. ' + 
+    'Es wurde für die Sozialarbeiter an dem ',
+    whatFor1B:  ' gedacht, um ihnen zu helfen, die Veränderungen von den ' + 
+    'Klientenunterstützungen vorherzusehen.',
+    whyImportantHeader:  'Warum ist dieses Werkzeug wichtig?',
+    why1:  'Ein Cliff Effect sich ereignet, als eine kleine Veränderung von der ' + 
+    'Situation eines Haushalts (z.B. eine oberflächliche Lohnerhöhung) den Wert der ' + 
+    'Unterstützung in einer unangemessenen Weise senkt.  Die Familie arbeitet, um ' + 
+    'ihre Einkommen sich zu verbessern, aber am Ende bekommt sie einen Nettoverlust, ' + 
+    'der sie eigentlich in eine schwerere Situation stellt.  Diese Cliff Effects ' + 
+    'hindern viele Familien daran, das Bekommen von Sozialhilfe eigentlich zu enden.',
+    why2:  'Die Cliff Effects sind schwer vorherzusehen.  Die Interaktionen zwischen ' + 
+    'dem Einkommen, der Nummer von Personnen im Haushalt, vielen verschiedenen anderen ' + 
+    'Krieterien, und den Auswirkungen der Programmen selbst haben Auswirkungen ' + 
+    'aufeinander in unerwarteten Weisen.  Wir suchen Methoden, um dieses Problem sich ' + 
+    'mit der Komplexität zu befassen, und die Familien ihre Situationen besser zu ' + 
+    'verstehen und vorherzusehen.',
```

Comment:
I would consider adding text for the links that are here in the English version. Even if the destinations are in English, they might still be useful if they can be automatically translated or if the user knows some English.

Reply:
I've added these as linkText1, linkText2, and linkText3.  These property names are basically just placeholders.